src/components/Input.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles((theme) => ({
+  textField: {
+    marginLeft: theme.spacing(1.5),
+    marginRight: theme.spacing(1.5),
+  },
+}));
+
+const Input = (props) => {
+  const { placeholder, name, value, onChange, required } = props;
+  const classes = useStyles();
+  return (
+    <TextField
+      required={required}
+      name={name}
+      value={value}
+      variant="outlined"
+      size="small"
+      placeholder={placeholder}
+      className={classes.textField}
+      onChange={onChange}
+    />
+  );
+};
```

Comment:
Wonderful job @sandaiiyahh! I love your use of the props and styling for this component. I checked it out on the production site, and everything looks great. 🚀